TALK.md

```diff
@@ -401,6 +399,48 @@ _container.requirements.general_ [Allocator-aware containers] N4687
 
 Containers use the `allocator_traits` template to get information about the allocator.
 
+---
+
+## OPTIONAL(REMOVE): Nullable Pointer - Exposition only
+
+```cpp
+template <typename T>
+concept NullablePointer = requires(T t) {
+    requires std::default_initializable<T>;
+    requires std::copy_constructible<T>;
+    requires std::assignable_from<T&, T>;
+    requires std::destructible<T>;
+    requires std::equality_comparable<T>;
+    requires std::equality_comparable_with<T, std::nullptr_t>;
+    requires std::swappable<T>;
+    { T(nullptr) } -> std::same_as<T>;
+    { t = nullptr } -> std::same_as<T&>;
+};
+```
+
+---
+
+## OPTIONAL(REMOVE): Allocator (C++20/23) - Exposition only
+
+```cpp
+template <template <typename> class A, typename T>
+concept Allocator = requires(A<T> a, typename std::allocator_traits<A<T>>::size_type s, typename std::allocator_traits<A<T>>::pointer p) {
```

Comment:
Allocator concept from the named requirement: https://en.cppreference.com/w/cpp/named_req/Allocator

Demonstrates the use of allocator traits to provide a default pointer, const_pointer, size_type, difference_type when not provided in the allocator (as Bob discusses in his talk)